moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -578,17 +578,17 @@ bool ServoCalcs::internalServoUpdate(Eigen::ArrayXd& delta_theta,
   // Mark the lowpass filters as updated for this cycle
   updated_filters_ = true;
 
+  // compose outgoing message
```

Comment:
As mentioned above, you may be overwriting this commit:
https://github.com/ros-planning/moveit2/commit/fb94e78da7d9a16fee53ccc34fbfb2c7af2af5d0